doc/Requirements Specification/Parts/Application.lyx

```diff
@@ -355,7 +352,7 @@ name "/A40/"
 
 \end_inset
 
- Beagle can be used to verify a software's design and implementation.
+ Beagle can be used to verify design and implementation of a software.
```

Comment:
This is not correct either ;)
you just changed the order of the words and kept the error.
I recommend to keep the first version and just delete the "a" ("Beagle can be used to verify a software's design and implementation.")